app/components/form/ClearAndAddButtons.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+import { Button } from '~/ui/lib/Button'
+
+type ClearAndAddButtonsProps = {
+  addButtonCopy: string
+  onClear: () => void
+  onSubmit: () => void
+  isDirty: boolean
```

Comment:
It was already like this, but I'd rather call this `disableClear` because this component doesn't know or need to know anything about the semantics of the form it is called from.